### DISCUSSION TITLE
Custom Risks in the Individual Risk Categories are missing from the build

### DISCUSSION THREAD

**Discussion by DavidCheuk** [Threagile/threagile#103] [Q&A]
Hi, I am the guy who issued this https://github.com/Threagile/threagile/issues/102, and I am now trying to add the missing logic to deal with the issue.  However, I find that the behavior of the binary I built from the source code here, is different from the docker version in a way that the  Custom Risks in the Individual Risk Categories are missing from the report generated from the binary I built where the report created from the docker version show them.  Did I miss something?  I originally thought my changes introduced the bug, but lately, I downloaded the repo in another location and tried to build it without any changes,  it proves that the code here does not include the custom risks.

**Comment by ezavgorodniy** [comment]
@joreiche made a changes to use custom risks defined as a script (something like this https://github.com/Threagile/threagile/blob/master/pkg/risks/scripts/accidental-secret-leak.yaml)
